src/towncrier/test/test_build.py

```diff
@@ -1625,34 +1623,45 @@ def test_invalid_fragment_name(self, runner):
         with open("foo/newsfragments/feature.124", "w") as f:
             f.write("This has the issue and category the wrong way round")
 
-        result = runner.invoke(
-            _main, ["--draft", "--date", "01-01-2001", "--version", "1.0.0"]
-        )
+        result = runner.invoke(_main, ["--draft"])
         self.assertEqual(1, result.exit_code, result.output)
         self.assertIn("Invalid news fragment name: feature.124", result.output)
 
     @with_project(
         config="""
         [tool.towncrier]
         package = "foo"
-        template = "foo/newsfragments/template.jinja"
+        template = "foo/newsfragments/template.j2"
+        ignore = ["placeholder-to-trigger-strict-checks.txt"]
```

Comment:
@aviramha  the extra file was ignored by default for `towncrier build` when "ignore" was not defined.

I am not sure how this tests was failing on your side.

Without adding the `ignore` config, the tests was passing , even with the code from 24.7.0